front/src/applications/stdcm/utils/formatSimulationReportSheet.ts

```diff
@@ -67,33 +79,45 @@ export function addMinutesToTime(
   return `${String(finalHour).padStart(2, '0')}:${String(finalMinutes).padStart(2, '0')}`;
 }
 
-function getTimeAtPosition(
-  trainPosition: number,
-  trainPositions: number[],
-  trainTimes: number[],
-  trainDepartureHour: number,
-  trainDepartureMinute: number
-): string {
+/**
+ * @property positions List of positions of a train in mm
+ * @property times List of times in milliseconds corresponding to the train positions
+ * @property departureHour Hour of the train departure (24h format)
+ * @property departureMinute Minute of the train departure
+ */
+interface TrainSimulationProps {
```

Comment:
The name `TrainSimulationProps` is misleading as it implies these are React component props, which they are not. What do you think about renaming it to `TrainSimulation` ? Additionally, please use `type` instead of `interface` in according to our guidelines.
```suggestion
type TrainSimulation = {
```

Reply:
Sure! I was not satisfied with the name either, but I feel TrainSimulation implies this contains more than it does. I've considered TrainSimulationPartial, TrainSimulationSimplified, TrainSimulationSpacetime, TrainTimesPositions, TrainKinematics, and others, but I'm not sure I'm convinced by any. Perhaps you're right and simply TrainSimulation works best.

Btw, why do we prefer type over interface? And are these guidelines written somewhere? If so I've been making prs blind this whole time haha

Reply:
I think `TrainTimesPositions` is a good name, it's descriptive enough.

Yes you can find the guidelines [here](https://github.com/osrd-project/osrd-confidential/issues/623).

I think we decided to use `type` over `interface` because `interface` is best for defining contracts (like module APIs, public methods, class shapes) and fits better in OOP paradigm where we define behaviors. while `type` is better for simple data structures

Reply:
Thanks a lot! There were definitely some conventions I was not aware of.